systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -358,6 +368,63 @@ public static String getImageValueFromCO(String name) {
         }
         return "";
     }
+    /**
+     * Log actual status of custom resource with pods.
+     * @param customResource - Kafka, KafkaConnect etc. - every resource that HasMetadata and HasStatus (Strimzi status)
+     */
+    public static <T extends HasMetadata & HasStatus> void logCurrentResourceStatus(T customResource) {
+        List<String> printWholeCr = new ArrayList<>(asList(KafkaConnector.RESOURCE_KIND, KafkaTopic.RESOURCE_KIND, KafkaUser.RESOURCE_KIND));
```

Comment:
`printWholeCR`? :)

Reply:
:D yes better :) thanks for review :)

Reply:
use factory method ->  `Arrays.asList()` instead  of explicitly calling constructor `new ArrayList<>`